packages/components/src/docs/General/11-FAQ.mdx

```diff
@@ -78,12 +78,43 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
 
 <sd-accordion>
   <h3 className="text-base sb-unstyled" slot="summary">
```

Comment:
Can't you just wrap EVERYTHING inside a `sb-unstyled` so we don't have to repeat that again and again?